src/Package/Impl/Help/HelpVisualComponent.cs

```diff
@@ -38,10 +38,13 @@ internal sealed class HelpVisualComponent : IHelpVisualComponent {
         /// unchanged and only replace browser that is inside it.
         /// </summary>
         private readonly ContentControl _windowContentControl;
+        private readonly IVignetteCodeColorBuilder _codeColorBuilder;
         private IRSession _session;
         private WindowsFormsHost _host;
 
         public HelpVisualComponent() {
+            _codeColorBuilder = VsAppShell.Current.ExportProvider.GetExportedValue<IVignetteCodeColorBuilder>();
```

Comment:
Actually I can't. Tests create components via reflection 

```
        public ControlTestScript(Type type) {
            ControlWindow.Create(type);
        }
```

We probably should change this and pass some sort of initialization context